tests/test_aquacomputer.py

```diff
@@ -154,12 +184,32 @@ def test_d5next_get_status_from_hwmon(mockD5NextDevice, tmp_path):
     assert sorted(got) == sorted(expected)
 
 
-def test_d5next_set_fixed_speeds_not_supported(mockD5NextDevice):
-    with pytest.raises(NotSupportedByDriver):
-        mockD5NextDevice.set_fixed_speed("fan", 42)
+def test_d5next_set_fixed_speeds_directly(mockD5NextDevice):
+    mockD5NextDevice.set_fixed_speed("fan", 50)
+    mockD5NextDevice.set_fixed_speed("pump", 84)
 
-    with pytest.raises(NotSupportedByDriver):
-        mockD5NextDevice.set_fixed_speed("pump", 84)
+    fan_report, pump_report = mockD5NextDevice.device.sent
+
+    assert fan_report.number == 3
+    assert fan_report.data[0x40:0x43] == [0, 19, 136]  # 0, <5000>
+    assert pump_report.number == 3
+    assert pump_report.data[0x95:0x98] == [0, 32, 208]  # 0, <8400>
+
+
+def test_d5next_set_fixed_speeds_hwmon(mockD5NextDevice, tmp_path):
```

Comment:
It might be wise to also test whether the fallback to direct mode when the PWM attributes are missing.